### PR TITLE
refactor: Remove version from HTML

### PR DESCRIPTION
**Pull request by bennyn**
Since the introduction of the config file approach, there is no more need to read the "version" from our HTML code.

### CODE REVIEW

**Review by Yserz — APPROVED** [review]

File: src/script/util/Environment.ts [hunk 1]
```diff
@@ -62,11 +53,11 @@ export const Environment = {
     }
 
     if (doNotFormat) {
-      return getAppVersion();
+      return Config.getConfig().VERSION;
```
Comment (ffflorian) [inline src/script/util/Environment.ts]: There is no formatting anymore so we should only check in this function if we want to show the electron version or not.

File: src/page/template/meta.htm [hunk 2]
```diff
@@ -1,7 +1,6 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <meta name="geoip" country="{{country}}" />
-    <meta property="wire:version" version="{{@config.VERSION}}" />
```
Comment (ffflorian) [inline src/page/template/meta.htm:4]: Let's please keep this because it's quite handy to use.

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: src/page/template/meta.htm (hunk 2 above)
Comment (Yserz) [reply src/page/template/meta.htm:4]: I don't think this is a good way of checking for the application version. Also it's way easier to check for `window.wire.env.VERSION`.

Comment (ffflorian) [reply src/page/template/meta.htm:4]: I didn't mean to continue checking from here but to just leave it here for convenience.

Comment (Yserz) [reply src/page/template/meta.htm:4]: In general you can find the version also via `/version`, in any resource query below or in `window.wire.env.VERSION` via the console. So there are plenty of places to take the version from.

**Review by ffflorian — APPROVED** [review]

**Pull request merged**